project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php

```diff
@@ -20,6 +20,11 @@ protected function checkEntityAccess($op, $entity_type, $entity) {
     $resource_name = $this->getResourceName();
     $group_id = $this->request['group'];
     $wrapper = entity_metadata_wrapper('node', $group_id);
+    
+    if (empty($wrapper->c4m_og_status)) {
+      return TRUE;
```

Comment:
@ordavidil If the entity is not of a group, maybe I should return `FALSE` instead (= don't return object). What do you think?